tests/e2e/unified_test.go

```diff
@@ -337,8 +378,35 @@ func TestAllInSeries(t *testing.T) {
 	cancel()
 }
 
+
+func mustNotPost(t *testing.T, got string, normalizers ...func(s string) string) {
```

Comment:
Q: Why only POST?  Do the controllers make any requests to GCP when paused?

Reply:
that's a good point; I figured we may fetch the live state in places but let me explore more and get back to you.

Reply:
In my further testing I didn't see any issues with GETs